helpers/pick.js

```diff
@@ -0,0 +1,32 @@
+/** @license ISC License (c) copyright 2017 original and current authors */
+/** @author Ian Hofmann-Hicks (evil) */
+
+const curry = require('./curry')
+const isFoldable = require('../predicates/isFoldable')
+const isObject  = require('../predicates/isObject')
+const isString = require('../predicates/isString')
+
+function pickKeys(obj) {
+  return function(acc, key) {
+    if(!isString(key)) {
+      throw new TypeError('pick: Foldable of Strings is required for first argument')
+    }
+    return key && obj[key] !== undefined
+      ? Object.assign({}, acc, { [key]: obj[key] })
```

Comment:
No need to assign into a new `Object`. should just reuse accumulator and keep creation cost down.